### ISSUE TITLE
RetinaFace transfer learning

### ISSUE THREAD

**Issue by licksylick** [biubug6/Pytorch_Retinaface#177]
I want to tune pretrained model (backbone Resnet50) for detection faces with correctly/incorrectly worn face mask and faces without mask. Сan someone explain/help me with this?
